icu4c/source/i18n/smpdtfmt.cpp

```diff
@@ -2561,10 +2561,10 @@ SimpleDateFormat::parse(const UnicodeString& text, Calendar& cal, ParsePosition&
             if (btz != NULL) {
                 if (tzTimeType == UTZFMT_TIME_TYPE_STANDARD) {
                     btz->getOffsetFromLocal(localMillis,
-                        BasicTimeZone::kStandard, BasicTimeZone::kStandard, raw, dst, status);
+                        (UTimeZoneLocalOption)BasicTimeZone::kStandard, (UTimeZoneLocalOption)BasicTimeZone::kStandard, raw, dst, status);
                 } else {
                     btz->getOffsetFromLocal(localMillis,
-                        BasicTimeZone::kDaylight, BasicTimeZone::kDaylight, raw, dst, status);
+                        (UTimeZoneLocalOption)BasicTimeZone::kDaylight, (UTimeZoneLocalOption)BasicTimeZone::kDaylight, raw, dst, status);
```

Comment:
Same question about UCAL_TZ_LOCAL_DAYLIGHT_* values.

Reply:
Similar to above. In this case:

```
btz->getOffsetFromLocal(localMillis,
    UCAL_TZ_LOCAL_DAYLIGHT_FORMER, UCAL_TZ_LOCAL_DAYLIGHT_LATTER, raw, dst, status);
```